RELEASING.md

```diff
@@ -0,0 +1,60 @@
+# Releasing Federation
+
+This is a quick and opinionated set of commands for building and releasing a new version of the packages in this monorepo.
+
+#### Start a release PR
+
+1. `FEDERATION_RELEASE_VERSION=X.Y.Z` (e.g. `2.0.1` or `2.0.0-beta.3` or `2.0.0-preview.99`)
+1. `git fetch origin main:main` to fetch the latest `main` branch
+1. `git checkout -b "release-$FEDERATION_RELEASE_VERSION" main`
+1. Update `CHANGELOG.md` in each repo ([more info](#changelogs))
+1. `git add ':/*CHANGELOG.md'` to add only changelogs to the commit
+1. `git commit --message "Update CHANGELOGs for v$FEDERATION_RELEASE_VERSION release"`
+1. `git push -u origin "release-$FEDERATION_RELEASE_VERSION"`
+1. `open "https://github.com/apollographql/federation/compare/main...release-$FEDERATION_RELEASE_VERSION?quick_pull=1&title=Release+$FEDERATION_RELEASE_VERSION&template=APOLLO_RELEASE_TEMPLATE.md"` to open PR in Github
```

Comment:
I really like this. Can it exist as a shell script in the repo?

Reply:
Absosmurfly

Reply:
...but I'll do it as part of another set of changes https://github.com/apollographql/federation/issues/1642

Lemme know if that seems OK to you

Reply:
`echo` makes sense, as well as adding in some commands for `gh`